OpenEdXMobile/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -240,17 +241,17 @@ public String getEmail() {
     }
 
     private void showResetPasswordDialog() {
-        ResetPasswordAlertDialogFragment.newInstance(this,
-                getEmail()).show(getSupportFragmentManager(), null);
+        ResetPasswordDialogFragment.newInstance(getEmail()).show(getSupportFragmentManager(), null);
     }
 
     public void showEulaDialog() {
-        environment.getRouter().showWebViewActivity(this, getString(R.string.eula_file_link), getString(R.string.end_user_title));
+        environment.getRouter().showWebViewActivity(this, getString(R.string.eula_file_link),
+                getString(R.string.end_user_title));
     }
 
     // make sure that on the login activity, all errors show up as a dialog as opposed to a flying snackbar
     @Override
-    public void showAlertDialog(String header, String message) {
+    public void showAlertDialog(@Nullable String header,@NonNull String message) {
```

Comment:
There should be a space before the `@NonNull` annotation at the start of the second argument.